### PR TITLE
Stopwatch

### LINKED ISSUE #207: Stopwatch
### **STOPWATCH**

Create a **stopwatch application** with options to:
- **Start** timekeeping on it.
- **Stop** it at the current time.
- **Reset** it to 0.

The stopwatch should display the time since start in **_seconds : milliseconds_** format.

**Example:**

Screenshot (163)

**Hints:** JavaScript

### PR DESCRIPTION
**Pull request by RohanMalik2710**
resolved issue #207

### CODE REVIEW

File: web/stopWatch.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    if (tens > 99) {
+        seconds++;
+        appendSeconds.innerHTML = "0" + seconds;
+        
+        tens = 0;
+        appendTens.innerHTML = "0" + 0;
+    }
+
+    if (seconds > 9) {
+        append.seconds.innerHTML = seconds;
```
Comment (harshjohar) [inline web/stopWatch.js]: I think u intended to do this here.

```suggestion
        appendSeconds.seconds.innerHTML = seconds;
```

**Review by harshjohar — CHANGES_REQUESTED** [review]
Here are the problem with your code.

- Your timer is going faster than real clock, check this with a real stopwatch you can realise this with eyes too
- The stylesheet of this stopwatch is not same as in the issue (the background, button colours and other styles)

**Comment by RohanMalik2710** [comment]
Sir the thing is CSS is completely new to me. So, I have tried to create a stopwatch application with a basic interface only using HTML and JavaScript and have avoided CSS. That's why its interface seems a little empty compared to what is prescribed

**Comment by harshjohar** [comment]
Ok leave out CSS, but the speed of stopwatch is not proper.. it is faster than required fix it only

**Comment by RohanMalik2710** [comment]
> Ok leave out CSS, but the speed of stopwatch is not proper.. it is faster than required fix it only

Ok sir will fix it and revert back to you

**Comment by RohanMalik2710** [comment]
The timer is running on normal speed now + also I have also made another change that till 9 sec timer shows a "0" before the number ,for example, 4 is represented as 04. But after timer pass 10 seconds, the prequel "0" gets removed ,for example, 10 is represented as 10 instead of 010 as it was showing before. I think the issues are resolved now. Please check

**Review by ken1000minus7 — APPROVED** [review]
Lgtm

**Pull request merged**